recipes/qbs/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, tools
+import os
+
+
+class QbsConan(ConanFile):
+    name = "qbs"
+    description = "The Qbs build system"
+    topics = ("conan", "qbs", "build", "automation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qbs.io"
+    license = "LGPL-2.1-only", "LGPL-3.0-only", "Nokia-Qt-exception-1.1"
+    settings = {
+        "arch_build": ["x86_64"],
+        "os_build": ["Linux", "Macos", "Windows"]
+    }
+
+    def source(self):
+        pass
+
+    def build(self):
+        data = dict(**self.conan_data["sources"][self.version][str(self.settings.os_build)])
+        tools.download(**data, filename="creator.7z")
+        self.run("cmake -E tar x creator.7z")
+
+    def package(self):
+        if self.settings.os_build == "Linux":
+            self.copy("bin/qbs*", symlinks=True)
+            self.copy("bin/qt.conf", symlinks=True)
+            self.copy("lib/qtcreator/libqbs*", symlinks=True)
+            self.copy("lib/qtcreator/plugins/qbs/*", symlinks=True)
+            self.copy("lib/Qt/lib/libic*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Core*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5DBus*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Gui*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Network*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5PrintSupport*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Script*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Widgets*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Xml*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5XcbQpa*", symlinks=True)
```

Comment:
Is it required to copy all qt5 libraries?
Do we want this?
Aren't these here for `qt creator`? (of which you're downloading the binaries in `build` but are not copying the binaries)
On Linux, running `objdump -p` on `qbs` returns these non-system libraries:
`libqbscore.so.1.16` and `libQt5Core.so.5`

Reply:
> Is it required to copy all qt5 libraries?
> Do we want this?

No, it is not required to copy all qt5 libraries, but the ones listed above were necessary because this package has been compiled with config-ui enabled. That causes a dependency on libQt5Gui and a couple of others and of course the platform plugins. Maybe I have overlooked 2 or 3.

It is not ideal, but building Qbs from source in a Conan environment is not something I have time for.

Actually, we can just say that using `qbs config-ui` is forbidden and remove a lot of the libraries. I guess nobody would do that anyway in a Conan environment. I have updated the PR accordingly.

> Aren't these here for qt creator? (of which you're downloading the binaries in build but are not copying the binaries)

No, they are needed by `libqbscore` or one of its dependencies.

> On Linux, running objdump -p on qbs returns these non-system libraries:
libqbscore.so.1.16 and libQt5Core.so.5

Rather use `ldd` to find out run-time dependencies (including transitive ones). This does still not print the required plugins though.

Reply:
Marking as resolved. The latest commit includes only the absolutely required libraries.